web/docs/tutorials/dev-excuses-app/adding-operations.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Adding operations
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Now you'll need to create two files: `actions.js` and `queries.js` in the `ext` folder.
+
+Let’s add `saveExcuse()` action to our `actions.js` file. This action will save the text of our excuse to the database. 
+
+```js title=".../ext/actions.js | Defining an action"
+export const saveExcuse = async (excuse, context) => {
+  return context.entities.Excuse.create({
+    data: { text: excuse.text }
+  })
+}
+```
+
```

Comment:
We don't explain much here, for example what is the relationship between the wasp declaration of query and query itself, what is the context in action, ... .
I believe we explain this a bit more in the TodoApp (do we?)?
Do you think that might be confusing to people? Or do you think not explaining too much here is a benefit?

Reply:
I've made an overview in the previous section about operations and linked To-Do app tutorial if they'd like to know more